tests/target/match.rs

```diff
@@ -32,7 +32,16 @@ fn foo() {
             meh,
 
         // Test that earlier patterns can take the guard space
-        (aaaa, bbbbb, ccccccc, aaaaa, bbbbbbbb, cccccc, aaaa, bbbbbbbb, cccccc, dddddd) |
+        (aaaa,
+         bbbbb,
+         ccccccc,
+         aaaaa,
+         bbbbbbbb,
+         cccccc,
+         aaaa,
+         bbbbbbbb,
+         cccccc,
+         dddddd) |
```

Comment:
I think we should not apply the inner width heuristic to patterns - there is more motivation to keep them on one line.